packages/web-pkg/src/event/bus.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { EventBus } from './bus'
+
+describe('EventBus', () => {
+  test('pubSub', () => {
```

Comment:
keep it for now, publish and subscribe is the only functionality available at the moment, bare minimum. This will be extended in the runtime refactoring and tests get more mature